packages/fxa-auth-server/lib/routes/validators.js

```diff
@@ -292,6 +293,44 @@ module.exports.activeSubscriptionValidator = isA.object({
   cancelledAt: isA.alternatives(isA.number(), isA.any().allow(null)),
 });
 
+// This is a Stripe subscription object with latest_invoice.payment_intent expanded
+module.exports.susbscriptionsSubscriptionExpandedValidator = isA
+  .object({
+    id: isA.string().required(),
+    object: isA.string().allow('subscription').required(),
+    latest_invoice: isA
+      .object({
+        id: isA.string().required(),
+        object: isA.string().allow('invoice').required(),
+        payment_intent: isA
+          .object({
+            id: isA.string().required(),
+            object: isA.string().allow('payment_intent').required(),
+            client_secret: isA.string().required(),
+          })
+          .unknown(true)
+          .required(),
+      })
+      .unknown(true)
+      .required(),
+  })
+  .unknown(true);
+
+module.exports.susbscriptionsInvoicePIExpandedValidator = isA
```

Comment:
Oh, missed this on first skim: `susbscriptionsInvoicePIExpandedValidator` should probably be `subscriptionsInvoicePIExpandedValidator`?